### PR TITLE
added new readme 

### PR DESCRIPTION
**Pull request by iamfortune**
Added a new Readme draft in markdown format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,82 @@
-# Welcome 👋
+# GitHub Working Draft
 
+# Intro
 
-Thank you for using Storyblok and for taking the time to help us make it better! If you have found a bug or have a suggestion for a new feature, we would love to hear from you. 
+Storyblok is a visual CMS that works with any framework. Integrate with any app or frontend or framework. Publish anywhere with an API-first backend. Storyblok’s industry-leading Visual Editor makes it easy for marketing teams to independently create, edit, and collaborate.
```
Comment (manuelschroederdev) [inline README.md:5]: Can we link "Visual Editor" to https://www.storyblok.com/docs/editor-guides/visual-editor please?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,82 @@
...
+# Intro
 
-Thank you for using Storyblok and for taking the time to help us make it better! If you have found a bug or have a suggestion for a new feature, we would love to hear from you. 
+Storyblok is a visual CMS that works with any framework. Integrate with any app or frontend or framework. Publish anywhere with an API-first backend. Storyblok’s industry-leading Visual Editor makes it easy for marketing teams to independently create, edit, and collaborate.  
 
-## Looking for the Storyblok CLI?
+# Getting Started
 
-The Storyblok CLI is now in it's own separate repository. You can find it here: <https://github.com/storyblok/storyblok-cli>
+- [https://www.storyblok.com/docs/guide/getting-started](https://www.storyblok.com/docs/guide/getting-started) → this should be replaced by the new Quickstart page once it’s live
```
Comment (manuelschroederdev) [inline README.md:9]: Please change the link title to "Get started with your favorite technology".
And remove the internal note ("this should be replaced...").

File: README.md [hunk 3]
```diff
@@ -1,17 +1,82 @@
...
+- **Internationalization**: Create personalized content experiences via built-in folder-level and field-level internationalization
+- **Omnichannel Experiences**: Storyblok’s headless architecture enables developers to publish content across any digital channel via APIs.
+
+# Resources
+
+- [Tutorials](https://www.storyblok.com/tutorials)
+- [FAQs](https://www.storyblok.com/faqs)
+- [Discord](https://discord.gg/jKrbAMz)
+- [Community](https://www.storyblok.com/community)
+- [Learning Hub](https://www.storyblok.com/docs)
```
Comment (manuelschroederdev) [inline README.md:68]: Can we also add the "Clients" as a section? js, ruby, python, php

**Review by manuelschroederdev — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (iamfortune) [reply README.md:68]: This will be [PHP SDK](link to php client)? @manuelschroederdev

**Review by manuelschroederdev — APPROVED** [review]

**Pull request merged**